lang/gcc-devel/Portfile

```diff
@@ -23,41 +30,17 @@ long_description    The GNU compiler collection, including front ends for \
 
 homepage            https://gcc.gnu.org/
 
-# Primary releases
-master_sites        https://ftpmirror.gnu.org/gcc/gcc-${version}/ \
-                    https://mirror.its.dal.ca/gnu/gcc/gcc-${version}/ \
-                    https://mirrors.kernel.org/gnu/gcc/gcc-${version}/ \
-                    https://www.mirrorservice.org/sites/ftp.gnu.org/gnu/gcc/gcc-${version}/ \
-                    https://ftp-stud.hs-esslingen.de/pub/Mirrors/ftp.gnu.org/gcc/gcc-${version}/ \
-                    https://mirror.yongbok.net/gnu/gcc/gcc-${version}/ \
-                    http://mirror.koddos.net/gcc/releases/gcc-${version}/ \
-                    ftp://ftp.gwdg.de/pub/linux/gcc/releases/gcc-${version}/ \
-                    ftp://gcc.ftp.nluug.nl/mirror/languages/gcc/releases/gcc-${version}/ \
-                    ftp://gcc.gnu.org/pub/gcc/releases/gcc-${version}/ \
-                    gnu:gcc/gcc-${version}
-# snapshots and RC candidates
-master_sites-append https://mirror.koddos.net/gcc/snapshots/${version}/ \
-                    https://bigsearcher.com/mirrors/gcc/snapshots/${version}/ \
-                    ftp://ftp.funet.fi/pub/mirrors/sources.redhat.com/pub/gcc/snapshots/${version}/ \
-                    ftp://gcc.gnu.org/pub/gcc/snapshots/${version}/
-
-distname            gcc-${version}
-use_xz              yes
-
-checksums           rmd160  df883a558afbdacac2152432873e2fd74c3c7125 \
-                    sha256  dd59afe340edc66b62626117790f0310d088b52d350118e48c061c08a6422be1 \
-                    size    72628316
-
-depends_lib-append  port:cctools \
```

Comment:
chris was asking for all this to remain in the Portfile, but just be commented out so we can easily revert back to it when the arm bits go upstream

Reply:
Indeed, I missed that. Would you like me to update this pull request correcting this and the other issues you pointed out, or do you guys want to take it over from here?

Reply:
Please update the PR yourself. Also please try and squash commits when you do it, as we only want a single commit appearing in the git history.